pkg/provider/provider.go

```diff
@@ -91,6 +93,10 @@ type KeyVaultObject struct {
 	ObjectName string `json:"objectName" yaml:"objectName"`
 	// the filename the object will be written to
 	ObjectAlias string `json:"objectAlias" yaml:"objectAlias"`
+	// the key filename the secret object will be written to
+	ObjectKeyFile string `json:"objectKeyFile" yaml:"objectKeyFile"`
+	// the cert filename the secret object will be written to
+	ObjectCertFile string `json:"objectCertFile" yaml:"objectCertFile"`
```

Comment:
Can you add these new fields in docs? 
https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/website/content/en/getting-started/usage/_index.md?plain=1#L64

Reply:
updated